app/src/services/mongo/MongoDBConnection.ts

```diff
@@ -1,85 +1,123 @@
-import mongoose, {Mongoose} from 'mongoose';
+import mongoose, { Mongoose } from 'mongoose';
 import ApplicationException from '../../exceptions/ApplicationException';
-import {getLogger} from '../../logger';
-import {injectable} from 'inversify';
-import Database, {DatabaseConfig} from '../../config/database';
+import { getLogger } from '../../logger';
+import { inject, injectable } from 'inversify';
+import Database from '../../config/database';
 
 @injectable()
 export default class MongoDBConnection {
     private static readonly logger = getLogger('DatabaseConnection');
-    private static isConnected = false;
-    private static connecting = false;
-    private static db: Mongoose;
-    private static dbConfig: Database;
 
-    constructor(dbConfig: Database) {
-        MongoDBConnection.dbConfig = dbConfig;
+    /**
+     * @type {boolean} true if connected successfully
+     */
+    private isConnected = false;
+
+    /**
+     * @type {boolean} true if currently connecting
+     */
+    private isConnecting = false;
+
+    /**
+     * @type{Mongoose} mongoose instance
```

Comment:
Spacing

Reply:
What is wrong here? I didn't get any errors

Reply:
Never mind